theia/operations/panoptes_operations/upload_subject.py

```diff
@@ -86,6 +90,58 @@ def _create_subject_set(self, project_id, subject_set_name):
 
         return subject_set
 
+    ## TODO: METHODS TO DELETE ONLY FOR FLOATING FORESTS DEBUG
+
+    def upload_mask_tiles(self, path_example, scope):
+        masked_tile_location =  path.dirname(path_example) + "_interstitial_products"
+        masked_tiles =  [masked_file for masked_file in os.listdir(masked_tile_location) if (path.isfile(path.join(masked_tile_location, masked_file)) and masked_file.endswith('.png'))]
+
+        masked_subject_set = self._create_subject_set(self.project.id, scope.name_subject_set() + ' masks')
+
+        for masked_filename in masked_tiles:
+            try:
+                masked_file_path = path.join(masked_tile_location, masked_filename)
+                metadata = {'mask_type': masked_filename}
+    
+                new_subject = self._create_subject(self.project.id, masked_file_path, metadata=metadata)
+                masked_subject_set.add(new_subject)
+            except Exception as masked_tile_upload_err:
+                print(masked_tile_upload_err)
+                pass
+
+
+
+    def upload_rejected_tiles(self, path_example, scope):
+        rejected_tile_location =  path.dirname(path_example) + "_interstitial_products/rejected"
+    
+        rejected_tiles = [rejected_file for rejected_file in os.listdir(rejected_tile_location) if (path.isfile(path.join(rejected_tile_location, rejected_file)) and rejected_file.endswith('.png'))]
```

Comment:
need to us `os.listdir` as opposed to importing `listdir` from `os` so that we can mock `os.listdir` in tests. (if referenced as `listdir` tests will fail; possibly something weird with `pytest` 🤷‍♀️ )

Reply:
Yep, this is an idiosyncracy of the way pytest locates modules 😂 . I'm OK with it!